### ISSUE TITLE
After restoring wallet from mnemonics and creating backup, the process is not finished

### ISSUE THREAD

**Issue by elvissio** [digitalbitbox/bitbox-wallet-app#814]
When you set up a new wallet on BitBox02 with mnemonic seed, and then create a backup on microSD card the procedure is not finished properly.

When selecting create backup a BitBoxApp asks to unlock the device. After unlocking the device the process is finished there (device idle or waiting the App) but nothing happens in the BBApp. The orange button "Check backup" is inactive.

There should be a message saying something like: "Congrats! You just created a backup and you should double check it in the next steps." or show the orange button "Check backup" as active.

If you go back (Back button is active) and then again to "Manage backups" all is as expected and the backup is visible.

**Comment by benma** [comment]
The issue is not with the setup process, but that the backup page is not refreshed after creating a backup. This is fixed in https://github.com/digitalbitbox/bitbox-wallet-app/pull/816

**Comment by Stadicus** [comment]
I did the same today when I wiped my BitBox02 and recreated the wallet from a mnemonic seedphrase. From a user perspective it was quite strange, because:

* I already had an old backup with that seedphrase on the SD card, but the BitBox02 was named (e.g. "HODL stash")
* After reset, the device has the default name ("My BitBox"?)
* When creating the backup, it shows the "HODL stash" backup, asks on the device if this name is correct ("My BitBox") and creates the backup.
* As the list is not refreshed, all I see is the old backup "HODL stash" with the old date, no new backup
* Going back and into Backup mgmt again, the "HODL stash" backup is gone and the "My BitBox" backup is shown (correct, as it's the same seedphrase)

A quick refresh directly after the backup creation would help here.
